test/compilable/imports/pkg16044/sub/package.d

```diff
@@ -0,0 +1,2 @@
+int test1;
```

Comment:
Might as well give them unique names (to ensure that it finds the members of this module)